new traitor mask: biochemist (#171)
EphemeralSpace/docs: src/design/secret-identities/traitor/biochemist.md

# Biochemist

{{#template ../../../templates/unimplemented.md }}

> **Name:** Biochemist
>
> **Organization:** [Traitor](../traitors.md)
>

> > **Description:** Throw your gas grenades in to medbay to cause panic and disease. Die when you inevitably get caught after doing something so obvious. Revive anyways thanks to your cryojunkie implant.

Reviewer: ```suggestion
> **Description:** Throw your gas grenades into medbay to cause panic and disease. Die when you inevitably get caught after doing something so obvious. Revive anyways thanks to your cryojunkie implant.
```

>
> **Objectives:** The same ones every traitor gets
>

> > *Okay guys, I'm going to release miasma in to medbay. Please wear a biosuit. Please. Please read the traitor radio so you don't die. I'm begging you. I don't even have any other weapons.*

Reviewer: ```suggestion
> *Okay guys, I'm going to release miasma in medbay. Please wear a biosuit. Please. Please read the traitor radio so you don't die. I'm begging you. I don't even have any other weapons.*
```

## Concept

The Biochemist is a traitor with a supportive-harasser playstyle, with some fun integration in to atmos and deniability coverage for some other existing secret identities.

## Abilities

> The Biochemist recieves an innate anti-death cryohusk implant, same as the one the [Cryojunkie](../crew/cryojunkie.md) recieves.

Reviewer: ```suggestion
## Abilities

The Biochemist receives an anti-death cryohusk implant in their cache, identical to the one that the [Cryojunkie](../crew/cryojunkie.md) receives.
```

In addition to this, the Biochemist also has the following items in their cache:
- 2 neurotoxin gas grenades
- 2 miasma gas grenades

## Gameplay

Enabled by their gas grenades and cryohusk implant, the Biochemist is able to adopt a unique playstyle focused around passive destruction, support, and planning. They mirror infiltrator [infiltrator,](../traitor/infiltrator.md) in this sense; they are offensively weak, but can accomplish a lot on their own (and for their team!) through non-standard means.

> On a more meta/simulationist level, the Biochemist adds a completely new method of attack to traitors in the form of atmospherics-based area denial, gives deniability to the [Cryojunkie,](../crew/cryojunkie.md) and acts as another more supportive archetype that is currently basically entirely absent out of the current traitor lineup besides [infiltrator.](../traitor/infiltrator.md)

Reviewer: ```suggestion
Enabled by their gas grenades and cryohusk implant, the Biochemist is able to adopt a unique playstyle focused around passive destruction, support, and planning. They mirror [infiltrator](infiltrator.md) in this sense; they are offensively weak, but can accomplish a lot for their team through non-standard means.

On a more meta/simulationist level, the Biochemist adds a completely new method of attack to traitors in the form of atmospherics-based area denial, gives deniability to the [Cryojunkie](../crew/cryojunkie.md), and is an entry in the supportive archetype that is currently only represented by [infiltrator](infiltrator.md).
```
